Makefile

```
@@ -128,8 +128,8 @@ deploy-stable:
 	$(call list-running-pods,$(CLUSTER_GSLB1))
 	$(call list-running-pods,$(CLUSTER_GSLB2))
 
-.PHONY: upgrade-candidate
-upgrade-candidate: ## Upgrade k8gb to the test version on existing clusters
+.PHONY: deploy-semver
+deploy-semver: ## Upgrade k8gb to the test version on existing clusters
```

```suggestion
deploy-test-version: ## Upgrade k8gb to the test version on existing clusters
```
I might be missing something but `deploy-semver` is a bit confusing target name here

---

The main idea for naming is we operate with `stable`[[1](https://github.com/k8gb-io/k8gb/blob/master/Makefile#L122)], [[2](https://github.com/k8gb-io/k8gb/blob/master/.github/workflows/upgrade-testing.yaml#L50)] [[3](https://github.com/k8gb-io/k8gb/blob/master/Makefile#L70)] and `semver` [[4](https://github.com/k8gb-io/k8gb/blob/master/Makefile#L67)] terms. and already use `stable`. I was trying to keep same semantics. 

Maybe  can rename both (deploy-stable, deploy-semver)

---

Agreed, using the word `semver` in this context is quite misleading. Semantic versioning talks about the versioning scheme, when to raise each component of that `x.y.z` triplet. So even the `stable` actually deploys a version (latest released in dockerhub) that conforms the semver.

 The makefile also operate with word "candidate", this is much closer to what it is, imho. Or `test-version` or `latest` vs `latest-released`(=stable).

The version/container tag it actually produces looks like `x.y.z-gitsha` and from what I've seen it's pretty much the same semantics as the `x.y.z-SNAPSHOT` versions from the Java world